code_block_timer/tests/test_timer.py

```diff
@@ -34,9 +35,31 @@ def test_stuff(self):
                     z += i
         self._verifyEvents(run_id, ['test', ] + ["test:{}".format(x) for x in iterations])
 
+    @mock.patch('code_block_timer.storage.TimingDataStorage')
+    def test_decorator(self, mock_class):
+
+        store = mock.Mock()
+        mock_class.return_value = store
+        store.run_id.return_value = 45
+
+        @code_block_timer('decorator_test', db_name=self.db_name)
```

Comment:
Actually, it seems that it's possible to make a method acts as both a context manager *and* a decorator. I'll put that on the TODO list.